config/locales/en.yml

```diff
@@ -52,9 +52,9 @@ en-GB:
       wrong_office: You are not authorized to manage this user.
   benefit_checks:
     :yes:
-      heading: '&#10003; &nbsp; The applicant is receiving the correct benefits'
+      heading: '✓ &nbsp; The applicant is receiving the correct benefits'
```

Comment:
I think we should consider adding the tick and cross using CSS, but that's not important now.